docs/Required-checks.md

```diff
@@ -0,0 +1,45 @@
+## Checks for the `master` branch
+
+Legend:
+* ✅ Required check.
+  Merges disallowed unless this job is successful.
+* 👉 Pre-requisite required check.
+  A required check depends on this job.  A job with a required check could not pass without this passing.
+* ⛔ No required check and not a pre-requisite.
+
+* Check Markdown links / markdown-link-check (push) ⛔
```

Comment:
emojis in on the left side would be more visible I think

Reply:
Done!